### DISCUSSION TITLE
[BUG - Update-Module -Name "PnP.Powershell" returning an error - PnP.Powershell was not installed by using Install-Module so it cannot be updated

### DISCUSSION THREAD

**Discussion by hsamdani12** [pnp/powershell#1759] [Q&A]
I'm currently on version 1.8.0 for PnP.Powershell.

When I run the following command - 
Get-Module *PnP* -ListAvailable | Select-Object Name,Version | Sort-Object Version -Descending

This is what I get  -
Name           Version
----           -------
PnP.PowerShell 1.8.0  
PnpDevice      1.0.0.0

I'm trying to update to latest version using the following command -
Update-Module -Name "PnP.Powershell" -RequiredVersion 1.10.0 -Force

### Expected behavior 
I'm sure I used Install-Module to install the PnP. Powershell v 1.8.0.  I should be able to update it to latest version of PnP.Powershell

### Actual behavior 
I get the following error when doing an upgrade -
Update-Module : Module 'PnP.Powershell' was not installed by using Install-Module, so it cannot be updated.
At line:1 char:1
+ Update-Module -Name "PnP.Powershell" -RequiredVersion 1.10.0 -Force
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (PnP.Powershell:String) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : ModuleNotInstalledUsingInstallModuleCmdlet,Update-Module

### Steps to reproduce behavior
Please include complete script or code samples in-line or linked from [gists](https://gist.github.com/)

### What is the version of the Cmdlet module you are running?
PnP.PowerShell 1.8.0 

### Which operating system/environment are you running PnP PowerShell on?
- [x ] Windows

**Comment by kdubious** [comment]
So how do we update?

**Comment by AndersRask** [comment]
You can always use install-module -force

man. 11. sep. 2023 16.27 skrev Kevin Welsh ***@***.***>:
